src/coreclr/vm/prestub.cpp

```diff
@@ -1853,6 +1860,11 @@ extern "C" PCODE STDCALL PreStubWorker(TransitionBlock* pTransitionBlock, Method
 {
     PCODE pbRetVal = NULL;
 
+    Module* pModule = NULL;
+    FrameWithCookie<ExternalMethodFrame> frame(pTransitionBlock);
```

Comment:
It does not look right to create a dummy `ExternalMethodFrame` just to get the return address. 

The fixups from R2R modules should come via `ExternalMethodFixupWorker`. Can we just use the module that is created there?